photon-lib/src/test/scala/com/linkedin/photon/ml/optimization/LBFGSBTest.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.optimization
+
+import breeze.linalg.DenseVector
+import com.linkedin.photon.ml.data.LabeledPoint
+import com.linkedin.photon.ml.normalization.{NoNormalization, NormalizationContext}
+import com.linkedin.photon.ml.test.{Assertions, CommonTestUtils}
```

Comment:
The breeze, spark, mockito, and testng imports should be in one import block together, and the photon imports should be in a second block below - take a look [here](https://github.com/databricks/scala-style-guide#imports).